wordpress-mobile/WordPress-iOS
# Use illustration on notes to prompt for Jetpack install

**Part 2 of** #8571 
In this PR we have updated the prompt to install Jetpack on the notifications screen to use the appropriate illustration.

**To test:**
1. Start with a clean slate.
2. Add a self hosted site: https://jurassic.ninja/create/?nojetpack
3. Go to Notifications and check that the illustration looks good.
4. Just for fun connect the site to Jetpack and see that everything works as expected.
5. Repeat on an iPad (simulator or device)

![simulator screen shot - iphone 8 - 2018-02-13 at 14 49 49](https://user-images.githubusercontent.com/5558824/36165092-34f432e6-10cd-11e8-8c7c-a700bcd884d5.png)
